packages/@okta/vuepress-site/guides/sign-into-mobile-app/check-on-startup/android/checkfortoken.md

```diff
@@ -37,20 +37,3 @@ client.introspectToken(client.getTokens().getAccessToken(),
     }
 );
 ```
```

Comment:
@FeiChen-okta  Does the developer always need to use the introspect approach or is it optional? What do you recommend? We can move any optional approaches into the library's Github readme.

Reply:
This is optional. Its if they want to check if the tokens are active.